<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisTickFormatDialog</class>
 <widget class="QDialog" name="AxisTickFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>X Axis Tick Format</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="3">
      <widget class="QToolButton" name="pbHelp">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QRadioButton" name="rbDefaultNoYear">
       <property name="text">
        <string>Default, no year
hh:mm:ss
dd.MM</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QRadioButton" name="rbDate">
       <property name="text">
        <string>Date only
dd.MM.yy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QRadioButton" name="rbDefault">
       <property name="text">
        <string>Default
hh:mm:ss
dd.MM.yy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QRadioButton" name="rbTime">
       <property name="text">
        <string>Time only
hh:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QLineEdit" name="leCustomFormat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>h:mm ap\ndd/MM/yyyy</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QRadioButton" name="rbCustom">
       <property name="text">
        <string>Custom:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbDefault</tabstop>
  <tabstop>rbDefaultNoYear</tabstop>
  <tabstop>rbTime</tabstop>
  <tabstop>rbDate</tabstop>
  <tabstop>rbCustom</tabstop>
  <tabstop>leCustomFormat</tabstop>
  <tabstop>pbHelp</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AxisTickFormatDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AxisTickFormatDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>leCustomFormat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
